qark/plugins/Manifest_File_Checks.py

```diff
@@ -58,66 +104,51 @@ def target(self, queue):
                 launch_mode_list.append(line)
 
             if any(re.findall(self.TASK_REPARENTING, line)):
-                PluginUtil.reportInfo(fileName, self.TaskReparentingIssue(fileName), res)
+                PluginUtil.reportInfo(file_name, task_reparenting(file_name), res)
+
+            if re.match(self.API_KEY_REGEX, line):
+                # Check if special character is present in the line. If "Yes, then ignore.
+                if re.match(self.SPECIAL_CHAR_REGEX, line):
```

Comment:
Change to `if not re.match(self.SPECIAL_CHAR_REGEX, line):` and take the code from the else and put it there. If you have a

```python
if statement:
    pass
else:
    do_stuff()
```

Then most of the time you can just negate the if and do the things you actually want instead of `pass`ing.